x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -137,6 +139,19 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
               ? TIMESTAMP
               : undefined;
 
+          // If we have a timestampOverride, we'll compute a runtime field that emits the override for each document if it exists,
+          // otherwise it emits @timestamp. If we don't have a timestamp override we don't want to pay the cost of using a
+          // runtime field, so we just use @timestamp directly.
+          const { aggregatableTimestampField, timestampRuntimeMappings } =
+            secondaryTimestamp && timestampOverride
```

Comment:
```suggestion
            secondaryTimestamp
```
If there's a secondary timestamp, `timestampOverride` must be `true`.

Reply:
@marshallmain True, but the type system doesn't seem to know that... 
<img width="1142" alt="image" src="https://user-images.githubusercontent.com/611653/183802689-c9635c13-fc7f-48b9-b679-31b3ef938c10.png">

Want me to be doing something else here?